### PR TITLE
BUGFIX: Respect site and dimensions in filter queries

### PR DESCRIPTION
**Pull request by paavo**
Fixes [punktDe#44](https://github.com/punktDe/form-persistence/issues/44)

### CODE REVIEW

**Comment by paavo** [comment]
hopefully you find time to review my bugfix this week @daniellienert  🙏

File: Classes/Controller/Backend/FormDataController.php [hunk 1]
```diff
@@ -134,7 +136,7 @@ public function previewAction(FormData $formDataEntry): void
      */
     public function deleteAction(FormData $formDataEntry): void
     {
-        $this->formDataRepository->removeByFormIdentifierAndHash($formDataEntry->getFormIdentifier(), $formDataEntry->getHash());
+        $this->formDataRepository->removeByFormIdentifierAndHash($formDataEntry->getFormIdentifier(), $formDataEntry->getHash(), $formDataEntry->getSitename(), $formDataEntry->getDimensionsHash());
```
Comment (daniellienert) [inline Classes/Controller/Backend/FormDataController.php:139]: Should also be named removeByFormProperties

**Review by daniellienert — CHANGES_REQUESTED** [review]
Thanks @paavo for the fix. Would you also adjust the remove method name?

File: Classes/Controller/Backend/FormDataController.php (hunk 1 above)
Comment (paavo) [reply Classes/Controller/Backend/FormDataController.php]: renamed ✔️

**Review by daniellienert — APPROVED** [review]

**Pull request merged**